ReactWindows/ReactNative/Views/TextInput/ReactPasswordBoxManager.cs

```diff
@@ -195,6 +196,17 @@ public void SetPlaceholder(PasswordBox view, string placeholder)
         }
 
         /// <summary>
```

Comment:
Quite a few apps use `placeholderTextColor`. I don't want the default behavior when trying to run these apps on UWP to be to throw an exception.  I'd prefer a yellow box warning that the prop is not available so the app can quietly work in Release mode and notify the developer of differences in debug mode.

---
In reply to: [103742136](https://github.com/ReactWindows/react-native-windows/pull/1038#discussion_r103742136) [](ancestors = 103742136)